app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -297,6 +296,7 @@ def pod_definition
             :command         => [
               "/usr/bin/image-inspector",
               "--image=#{options[:image_full_name]}",
+              "--scan-type=openscap",
```

Comment:
It might be better to add a ```--chroot``` flag. It is more secure that way. Can you please add it?